### PR TITLE
DOCS Update starter and Watea documentation for Bootstrap 4 and theme colour mixin usage

### PR DESCRIPTION
**Pull request by robbieaverill**
Issue: https://github.com/silverstripeltd/cc-issues/issues/298

### CODE REVIEW

File: docs/en/01_Working_with_projects/14_Using_the_Watea_theme.md [hunk 1]
```diff
@@ -119,4 +119,48 @@ For information on the rules and configuration around these linters, please see
...
++  @include theme-contrast-color("footer", "color");
+}
+```
+
+It's important to ensure that you use theme colour mixins whenever you are defining colours that are used in areas
+which may be modified by CMS colour customisations to ensure that the colours you define are also changed when their
+surrounding context changes, otherwise you may end up with text that is unreadable against various backgrounds.
+
+For more information on customising the colour configuration settings in the CMS, see the
+[cwp/agency-extensions documentation](https://github.com/silverstripe/cwp-agencyextensions/blob/2.2/docs/en/01_Features/ThemeColors.md).
```
Comment (robbieaverill) [inline docs/en/01_Working_with_projects/14_Using_the_Watea_theme.md]: cc @scott1702

**Comment by michalkleiner** [comment]
Is `yarn` the preferred way for everything CWP/SS rather than `npm` or `npx`?

**Comment by Cheddam** [comment]
@michalkleiner Yes - Yarn has been adopted across the full SilverStripe / CWP core at this point (along with Node 10.x from SilverStripe 4.4 onwards).

**Comment by michalkleiner** [comment]
Thanks @Cheddam. Is there any reasoning for this other than it was cool for a bit of time when npm didn't have lock files? Something along the lines why SS chose to use React for the CMS against Angular (or whichever other JS fw was the other contender)?

**Comment by robbieaverill** [comment]
@michalkleiner I think that one of our jobs is to choose appropriate tools for the job, implement them consistently, and promote them to other SilverStripe developers and users. Yarn would've been chosen at the time because it had locking which npm didn't (at the time), and we continue to adopt it across our supported module suite and promote its use.

In the case of this pull request - you're obviously welcome to use whatever build tooling you want when working on CWP projects. These docs are to suggest recommended approaches for people to get started with the themes, and using the same toolchain as the rest of SilverStripe is a good way to minimise confusion on that front.

**Comment by michalkleiner** [comment]
@robbieaverill I'm not questioning that and certainly didn't want to come across "in attack mode", I was just curious if the only reason was the lock file or there was something else. Agree that the discussion is out of the scope of this PR. Cheers

**Pull request merged**